codedog/chains/pr_summary/base.py

```
@@ -12,8 +12,7 @@
 from langchain.output_parsers import OutputFixingParser, PydanticOutputParser
 from langchain_core.output_parsers import BaseOutputParser
 from langchain_core.prompts import BasePromptTemplate
-from langchain_core.pydantic_v1 import Field
-from pydantic import BaseModel
+from pydantic import Field, BaseModel, ConfigDict
```

Ruff (0.8.2)

`pydantic.BaseModel` imported but unused

Remove unused import: `pydantic.BaseModel`

(F401)